backend/pick-git/src/main/java/com/woowacourse/pickgit/user/presentation/dto/response/SearchResponse.java

```diff
@@ -0,0 +1,23 @@
+package com.woowacourse.pickgit.user.presentation.dto.response;
+
+import com.woowacourse.pickgit.user.application.dto.response.UserSearchResponseDto;
+import java.util.List;
+import lombok.Builder;
+
+@Builder
+public class SearchResponse {
+
+    private List<UserSearchResponseDto> users;
+
+    public SearchResponse() {
+    }
+
+    public SearchResponse(
+        List<UserSearchResponseDto> users) {
+        this.users = users;
+    }
+
```

Comment:
```suggestion
    public SearchResponse(List<UserSearchResponseDto> users) {
        this.users = users;
    }

```

100자 초과가 아니라면 인라인으로 작성해도 될것같습니다.